web/src/copy_and_paste.ts

```diff
@@ -293,63 +313,71 @@ function get_end_tr_from_endc($endc) {
         // we can use the last message from the previous recipient_row.
         if ($endc.parents(".message_header").length > 0) {
             const $overflow_recipient_row = $endc.parents(".recipient_row").first();
-            return $overflow_recipient_row.prev(".recipient_row").last(".message_row");
+            return $overflow_recipient_row.prev(".recipient_row").last();
```

Comment:
Is this a functional change? I guess not, because the last item in a `recipient_row` is always a `message_row`; I'm just not sure that's guaranteed to be an invariant. 

```
        <div class="recipient_row" id="{{message_group_id}}">
            {{> recipient_row use_match_properties=../use_match_properties}}
            {{#each message_containers}}
                {{#with this}}
                {{> single_message use_match_properties=../../use_match_properties message_list_id=../../message_list_id}}
                {{/with}}
            {{/each}}
        </div>
```

But probably we meant to write `.filter(".message_row").last()`?

Reply:
Actually this is really quite confusing -- the comments think it's getting a `message_row`, but it's always been getting a `recipient_row`

Reply:
I don’t know what the intention was, but the [jQuery `.last()` method does not accept any arguments](https://api.jquery.com/last/).

Reply:
Yeah I'll let @N-Shar-ma clean this up, since this is ancient code but I think she has a good test setup for.